doc/daal4py/conf.py

```diff
@@ -29,9 +29,10 @@
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 #
-# import os
-# import sys
-# sys.path.insert(0, os.path.abspath('.'))
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath("../.."))
```

Comment:
Since I don't know this side, but the PR looks simple enough, small explanation here as a comment and I'll approve (given it makes sense).

Reply:
The docs are generated from the python docstrings through `autodoc`. That means it needs to be able to import the package in order to build the docs.

This line adds the folder where `daal4py` would be to the paths where python searches, assuming you built the extension beforehand. Looks like it's not working in the CI job "docs" though, but the same logic is used in sklearnex, and works if you try `make html` locally:
https://github.com/uxlfoundation/scikit-learn-intelex/blob/95bd1eabcca2e4d0565423be1dba3b58deaf8901/doc/sources/conf.py#L35